clients/redshift/redshift.go

```diff
@@ -91,7 +92,34 @@ WHERE
 }
 
 func (s *Store) Dedupe(fqTableName string) error {
-	return fmt.Errorf("dedupe is not yet implemented")
+	stagingTableName := fmt.Sprintf("%s_dedupe_staging_%.5d", constants.ArtiePrefix, rand.Intn(100_000))
```

Comment:
Use this instead? https://github.com/artie-labs/transfer/blob/a857a7fd9521bb14933270483279185444f81aa5/clients/redshift/writes.go#L14

That way if this fails, our GC will drop the table

Reply:
Is it possible for this to fail in a way it doesn't get cleaned up though since it's in a transaction?

Reply:
Probably not, still good to keep temp table names consistent though

Reply:
Will make the change in a follow up PR, want to clean up the temporary table name logic so that it can be reused.